contributing-guides/style-guide.md

```diff
@@ -104,40 +104,74 @@ Example:
+  - you exactly know they are required and the documentation just missing information about this
+- Use double quotes for placeholders where variable or another substitution occures.
+  In other cases use single quotes.
+
+> :bulb: Many shells have [word splitting][word_splitting] enabled by default so unquoted arguments may be splitted to
+several ones. But the main goal of Tl;Dr pages is to provide easy to read and simple in use pages but
+not to strictly follow POSIX shell syntax.
+
+[word_splitting]: https://mywiki.wooledge.org/WordSplitting
 
 ### Paths
-- Use `{{filename}}` rather than `{{file_name}}`.
-- For any reference to paths of files or directories,
-  use the format `{{path/to/<placeholder>}}`,
-  except when the location is implicit.
-- When the path cannot be relative,
-  but has to start at the root of the filesystem,
-  prefix it with a slash,
-  such as `get {{/path/to/remote_file}}`.
-- In case of a possible reference both to a file or a directory,
-  use `{{path/to/file_or_directory}}`.
-
-Extensions
-
-- If a particular extension is expected for the file, append it.
-  For example, `unrar x {{compressed.rar}}`.
-- In case a generic extension is needed, use `{{.ext}}`, but **only** if an extension is required.
-  For instance, in `find.md`'s example "Find files by extension" (`find {{root_path}} -name '{{*.ext}}'`)
-  using `{{*.ext}}` explains the command without being unnecessarily specific;
-  while in `wc -l {{file}}` using `{{file}}` (without extension) is sufficient.
+
+- Use `{{path/to/file}}` or `{{path/to/directory}}` placeholders for Unix-like command pages for file and directory paths respectively.
+  For Windows command pages use `{{path\to\file}}` or `{{path\to\directory}}` placeholders.
+- Where file and directory paths are both vaild use `{{path/to/file_or_directory}}` placeholder for Unix-like command pages
+  or `{{path\to\file_or_directory}}` for Windows command pages.
+- For absolute paths add leading slash (or backslash for Windows command pages) at the beggining of placeholder such
+  as `{{/path/to/file}}` or `{{/path/to/directory}}`.
+- Replace `file` and `directory` words from placeholders above with more descriptive ones
+  just when:
+  - documentation describes command argument semantics explicitly
+  - you exactly know placeholder meaning (maybe from source code) and the documentation just missing information about this.
+
+> :bulb: If some command is cross-platform then Unix-style path placeholders are preferred.
+
+### Extensions
+
+- Append `.ext` to the end of a file path placeholder just when:
+  - documentation requires extension explicitly
+  - you exactly know it is required (maybe from source code) and the documentation just missing information about this
+  - adding extension is an good practise described anywhere
+  
+  For example: `unrar x {{path/to/compressed.rar}}`
+- Replace `ext` word from placeholder above with one of concreete extensions
```

Comment:
This does not make grammatical sense, and ref the comment above I've unsure how to correct this.

Reply:
I don't even know what this is trying to say, whichi s probably why it's hard to correct. 🤔

Reply:
concreete → concrete

Is "`ext` word" referring to `.rar` in the line above?
Maybe: "Replace the extension in the example above (`rar`)"

No idea what a "concrete extension" is, so a reader won't either.

Reply:
I refer to some extension in placeholder. Not to `rar`.

Reply:
What does "some extension in placeholder" mean?

Reply:
For instance we have `engrapma {{path/to/archive.ext}}` code sample. It's better to replace `ext` with any concrete extension such as `tar` or `zip` instead of leaving example as is.

Reply:
Ahh I see, `{{placeholder}}` is what you're renaming token too. 🤔 
I wouldn't say I agree with this change, those are not placeholders, they're just user input.

For many commands, they have defined parameters to match the docs, not a placeholder that users should replace. Token fits better imo.

> It's better to replace ext with any concrete extension such as tar or zip instead of leaving example as is.

Sorry, but I'd also disagree with this. Or at least based on our current logic, I think it's better to keep things agnostic, unless it only supports `tar` files for example, we should probably be omitting the extension entirely. 🤔 

---

Regardless, if we do make this chance to the wording, then my suggestion does make sense. 

> Is "`ext` word" referring to `.rar` in the line above?
> Maybe: "Replace the extension in the example above (`rar`)"

So replace the extension, which in the example above happens to be `rar`. Though if you think it's unclear, then perhaps provide don't reference anything else at all.

Just describe it exactly how you described it me now.

> For instance, for the command `engrapma {{path/to/archive.ext}}`. It's better to replace use a concrete file extension like as `tar` or `zip` rather than something generic like `ext`.